### ISSUE TITLE
Get Started is kind of weird

### ISSUE THREAD

**Issue by doginthehat** [balderdashy/sails-docs#599]
Here's a more elaborate report following my tweet.

The Get Started page is a bit weird.

It's spends some much time explaining what node is and how to install it. 

The flow is like this:

1. Actual Get Started information.
2. Actually written: "Now, let's get Sails to do cool stuff."
3. Large amounts of underwhelming information about how to install node.js on every platform.
4. Some theoretical information about sails' broad architecture.
5. Links to external sites that aren't helping to get started either.

I can see some of the follow up information on get started is displayed once you preview the site after  `sails lift` but it might make more sense to show it here – and maybe move that node stuff in a separate page.

Some thing like this might be more useful

1. Get started as it is now (with maybe one short paragraph to link to a separate node installation page)
2. Adding a new model to the api
3. Introduction to blueprints
3. Configuring the database (info on default local db and one example of postgres or mongodb)
4. More info on default & custom routes
5. Policies

That might make a better "Get Started" page – IMHO of course :)

**Comment by sailsbot** [comment]
Thanks for posting, @doginthehat.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by doginthehat** [comment]
Gosh, you feel like you're being listened to when Mr Autobot is the only contributor to your bug report and writes back to say your issue is closed :pensive:

**Comment by brettimus** [comment]
Hey Ben! I'm sorry no one has responded. I'll say that I agree with your critique. If you make a PR with some changes, I'd be glad to work with you to review + merge.

Sadly, I've been out of touch due to a mountain of Other Things :tm: on my plate. Open source is hard :disappointed:.

**Comment by brettimus** [comment]
Namely, I agree that there's a lot of setup details that could be moved to other sections. I think the focus should be less on understanding how sails works out the gate and more on “let's get up and running like right nyaw”

**Comment by doginthehat** [comment]
Yay, human interaction :smile: !

I could see if I can find some time to put a PR together for this, though it would seem like a major overhaul of the docs as it would need some new sections added.

Would that likely conflict with other pending PRs ?

**Comment by mikermcneil** [comment]
@doginthehat  Thanks for the feedback!  I've heard she likes to be called [Miss](https://github.com/balderdashy/sails/issues/3423#issuecomment-169751072), but I'm not sure she actually plays by normal gender rules (being a bot and all).

Any ideas on how to make her message more welcoming?  (she's on [vacation](http://asksailsbot.tumblr.com/post/136824812101/new-years-resolutions-of-github-issues) right now so it's probably safe to talk about her behind her back)  The fact of the matter is that, despite sometimes being annoying, she's made huge improvements to our issue management workflow.

@brettimus Thanks for getting all up on this man.  

> “let's get up and running like right nyaw”

Totally agreed.  I think we should keep the philosophy & architecture stuff in (I've had numerous people quote from it when talking to them in person over the years, so I think that part is helpful), but I'm cool with linking elsewhere for the Node.js stuff.

btw, just pushed out http://blog.sailsjs.org/post/137189916152/updates-to-how-we-contribute-to-the-sails-docs

**Comment by brettimus** [comment]
:+1: dig the changes!

and @doginthehat my advice would be to open a PR with some small-ish changes and not worry about current open PRs. we can iterate from there and merge when we think it's a clear improvement over the status quo. 

i'm in the middle of insites and flying across the country 3x in the next 3 weeks, so responding might take me a few days. 

if ever i go silent for > 2 days, ping me on twitter (@rudeboot).

godspeed!

**Comment by mikermcneil** [comment]
@doginthehat see also https://github.com/balderdashy/sails-docs/pull/366

**Comment by sailsbot** [comment]
Thanks for posting, @doginthehat.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
